src/components/data/data-utils.js

```diff
@@ -67,7 +68,8 @@ export const getUserProjectForWorkspace = async workspace => (workspace && await
   workspace.workspace.googleProject :
   requesterPaysProjectStore.get()
 
-const isUri = datum => _.startsWith('gs://', datum) || _.startsWith('dos://', datum) || _.startsWith('drs://', datum)
+const isViewableUri = (datum, workspace) => (isGoogleWorkspace(workspace) &&
+  isGs(datum)) || isDrs(datum) || (isAzureWorkspace(workspace) && (isAzureUri(datum)))
```

Comment:
Style nit: This expression is kind of hard to read. Maybe it could be tidied up a bit (IDK if ESLint will accept this exact suggestion)?
```suggestion
const isViewableUri = (datum, workspace) =>
  (isGoogleWorkspace(workspace) && isGs(datum)) ||
  (isAzureWorkspace(workspace) && isAzureUri(datum)) ||
  isDrs(datum)
```

Reply:
Updated here!